vsintegration/src/FSharp.Editor/Hints/InlineParameterNameHints.fs

```diff
@@ -82,6 +82,12 @@ module InlineParameterNameHints =
         symbolUse.IsFromUse
         && symbol.DisplayName <> "(::)"
 
+    let isTooBasic (symbol: FSharpSymbol) = 
```

Comment:
Can this be renamed something like `isBuiltinKnownType` or `isBuiltinKnownSymbol`?